src/rgw/rgw_rest.cc

```diff
@@ -1550,8 +1550,8 @@ int RGWCompleteMultipart_ObjStore::get_params()
     return op_ret;
   }
 
-#define COMPLETE_MULTIPART_MAX_LEN (1024 * 1024) /* api defines max 10,000 parts, this should be enough */
-  std::tie(op_ret, data) = rgw_rest_read_all_input(s, COMPLETE_MULTIPART_MAX_LEN);
+  const auto max_size = s->cct->_conf->rgw_max_put_param_size;
+  std::tie(op_ret, data) = rgw_rest_read_all_input(s, max_size, false);
```

Comment:
ok. 1M is also the default for rgw_max_put_param_size, but it does make sense to allow configuration

but why change allow_chunked to false? especially when you expect large requests?

Reply:
oh...fixed